frontend/src/Review.js

```diff
@@ -8,64 +8,82 @@ class ReviewContainer extends React.Component {
         this.sendData = this.sendData.bind(this);
```

Comment:
I guess this whole file is still a TODO? I've already reviewed this far since the PR wasn't a draft, so I'm still gonna comment, but in the future it would be helpful to mark a PR draft as a WIP.

Reply:
This was addressed in subsequent commits.